pkg/schedule/config/config.go

```diff
@@ -63,6 +63,7 @@ const (
 	defaultRegionScoreFormulaVersion = "v2"
 	defaultLeaderSchedulePolicy      = "count"
 	defaultStoreLimitVersion         = "v1"
+	defaultPatrolRegionConcurrency   = 1
```

Comment:
What is the max value?

Reply:
Where is the most appropriate place to restrict this configuration? pd-ctl？

Reply:
I think we need to limit it in the server side.